metapool/data/abs_quant_sample_info_calc.yml

```diff
@@ -11,5 +11,7 @@
         "density_storage_liquid_g_ml": 1.12
       "231713":
         "density_storage_liquid_g_ml": 1.12
+      "236286":
+        "density_storage_liquid_g_ml": 1.10
```

Comment:
Out of scope for this PR, but if I understand this right, it seems like there is a lot of work necessary to add additional information about a lot, as changes to config files in a repository would necessitate a subsequent release and deploy as well?

Reply:
In theory yes but it isn't actually used that way.  There is one person on earth who runs the notebook that uses this config (aside from me, who wrote it) and that is MacKenzie.  She runs it locally on her machine and changes the config at will.  Suggestions for a better way to handle this are welcome.

Reply:
A simple REST endpoint in Qiita?

Reply:
If I understand you guys correctly there's a constant for the lab being defined in the back corner of a notebook that should probably be associated with the lab as a whole or with qiita study ids?

Reply:
I have to defer to @antgonza. I suppose that there might be many more constants that are defined in a notebook or by unwritten convention in the wetlab and maybe some or all of them are trivial, but I do find it concerning that there might not be proper provenance for them. I think with my limited knowledge of the domain I'm not qualified to judge.

Reply:
To be clear, they aren't defined in a notebook. They are defined in a yaml config file.

Reply:
I actually think this is a question for @mmbryant23; no? Anyway, IMOO we could store this yaml in some public place where is always retrieved and checked: like github or the ftp.

Reply:
Adjusting to source from GitHub is a good idea